docs/contributing/platform_api/app/auth.rst

```diff
@@ -0,0 +1,9 @@
+pulp.app.auth
```

Comment:
I think these need to be s/pulp/pulpcore/ right? Also the indention needs to line up too (iirc).